frontend/src/pages/Sessions/SessionsFeedV3/SessionQueryBuilder/components/QueryBuilder/QueryBuilder.tsx

```diff
@@ -1307,6 +1245,35 @@ enum QueryBuilderMode {
 	SEGMENT_UPDATE = 'SEGMENT_UPDATE',
 }
 
+const FORMAT = 'YYYY-MM-DDTHH:mm:00.000000000Z'
+const now = moment()
+const presetOptions = [
+	{
+		label: 'Last 15 minutes',
+		startDate: new Date(now.subtract(15, 'minutes').format(FORMAT)),
+	},
+	{
+		label: 'Last 60 minutes',
+		startDate: new Date(now.subtract(60, 'minutes').format(FORMAT)),
+	},
+	{
+		label: 'Last 4 hours',
+		startDate: new Date(now.subtract(4, 'hours').format(FORMAT)),
+	},
+	{
+		label: 'Last 24 hours',
+		startDate: new Date(now.subtract(24, 'hours').format(FORMAT)),
+	},
+	{
+		label: 'Last 7 days',
+		startDate: new Date(now.subtract(7, 'days').format(FORMAT)),
+	},
+	{
+		label: 'Last 30 days',
+		startDate: new Date(now.subtract(30, 'days').format(FORMAT)),
+	},
+]
+
```

Comment:
could we make another constant here
`const defaultPreset = presetOptioons[5]` and use that below?